packages/gatsby-theme-docs/src/modules/self-learning/components/profile-modal.tsx

```diff
@@ -228,17 +228,21 @@ const ProfileModal = () => {
             onBlur={formik.handleBlur}
           />
           {profile?.global_account_name ? (
-            <Label>
-              Your verified company association is{' '}
-              <b>{profile.global_account_name}</b> with id{' '}
-              <b>{profile.global_account_id}</b> <br />
-              <Link
-                isExternal
-                to={`mailto:${mailToData.email}?subject=${mailToData.company_association_verify_request.subject}&body=${mailToData.company_association_verify_request.body}`}
-              >
-                Please contact us to change it.
-              </Link>
-            </Label>
+            <SpacingsStack>
```

Comment:
Is this an unrelated fix?

Reply:
unrelated to the ticket purpose but relevant